apps/angular/projection/src/app/component/city-card/city-card.component.ts

```diff
@@ -1,13 +1,54 @@
 import { Component, OnInit } from '@angular/core';
+import { CityStore } from '../../data-access/city.store';
+import {
+  FakeHttpService,
+  randomCity,
+} from '../../data-access/fake-http.service';
+import { CardType } from '../../model/card.model';
+import { City } from '../../model/city.model';
+import { CardComponent } from '../../ui/card/card.component';
+import { ListItemComponent } from '../../ui/list-item/list-item.component';
 
 @Component({
   selector: 'app-city-card',
-  template: 'TODO City',
+  template: `
+    <app-card
+      [list]="cities"
+      [type]="cardType"
+      (add)="addNew()"
+      customClass="bg-light-red">
+      <img
+        src="assets/img/student.webp"
+        alt="Placeholder for the student card"
+        width="200px" />
+      <ng-template #rowRef let-city>
```

Comment:
#rowRef is a magic string, try to [avoid this](https://softwareengineering.stackexchange.com/a/365344).

Reply:
Hi @svenson95 ,

 How to get rid of this magic string ? Any examples on how to avoid this in content projection case ?

Reply:
Yes you just need to create a `Directive`, [here](https://medium.com/@thomas.laforge/ngtemplateoutlet-type-checking-5d2dcb07a2c6) is a nice article from @tomalaforge about this topic